pom.xml

```diff
@@ -2385,7 +2385,7 @@
         <!-- Identity REST API feature -->
         <identity.api.dispatcher.version>2.0.17</identity.api.dispatcher.version>
         <identity.user.api.version>1.3.38</identity.user.api.version>
-        <identity.server.api.version>1.2.210</identity.server.api.version>
+        <identity.server.api.version>1.2.211-SNAPSHOT</identity.server.api.version>
```

Comment:
```suggestion
        <identity.server.api.version>1.2.211</identity.server.api.version>
```

Reply:
This will be updated when the relevant component is released.